blazor/datagrid/in-line-editing.md

```diff
@@ -1269,3 +1269,24 @@ public class OrderDetails
 {% endtabs %}
 
 {% previewsample "https://blazorplayground.syncfusion.com/embed/LthINCtgBmWlAIwy?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+## Supported events for inline and dialog editing
+
+The Syncfusion Blazor DataGrid triggers the following events during `Inline` and `Dialog` editing:
+
+| Event | Description |
```

Comment:
Sequence of Events can be in order that execute. 

like 

Adding,Editing, Updateing, Deleting and Finally cancelling